index.d.ts

```diff
@@ -155,6 +155,22 @@ declare namespace normalizeUrl {
 		*/
 		readonly removeTrailingSlash?: boolean;
 
+		/**
+		Remove a sole `/` pathname in the output. This option is independant of `removeTrailingSlash`.
+
+		@default false
+
+		@example
+		```
+		normalizeUrl('http://sindresorhus.com/');
```

Comment:
Use `https`

Reply:
`http` is used throughout the docs, I'll update all those cases.

Reply:
Actually, it seems to module has a bias towards `http` in some cases, I guess I'll leave this for another PR and just update the protocol in my additions.

````js
> require(".")("//sindresorhus.com:80/")
'http://sindresorhus.com'
> require(".")("//sindresorhus.com:443/")
'http://sindresorhus.com:443'

Reply:
> it seems the module has a bias towards http in some cases

Yeah. I've been wanting to change everything possible to `https`.

Reply:
Done changing all new docs/tests to use https.